.gitignore

```
@@ -1 +1,24 @@
-.DS_Store
\ No newline at end of file
+# macOS system files
+.DS_Store
+**/DS_Store
+
+# Claude AI assistant files
+.claude/
+
+# Node.js
+node_modules/
```

Make subdir ignored too. Also, we can ignore `package-lock.json` since we're using pnpm.
```suggestion
node_modules/
**/node_modules/
```